setup.cfg

```diff
@@ -60,9 +60,7 @@ localstack =
 # required to actually run localstack on the host
 runtime =
     airspeed==0.5.19
-    # Use our "ext" version until this bug is fixed: https://github.com/awslabs/amazon-kinesis-client-python/issues/99
```

Comment:
The linked issue is technically not resolved yet, but following some discussion threads on the Web, the Maven JAR downloads should be more stable now. Hence, we should try going with the default `amazon_kclpy` library for now. In case we run into build issues again, we can update our forked version again.